docs/core/porting/unsupported-apis.md

```diff
@@ -0,0 +1,341 @@
+---
+title: Unsupported APIs on .NET Core
+description: Learn which APIs from the .NET Framework that always throw an exception on .NET Core.
+ms.date: 12/23/2019
+---
+# APIs that always throw exceptions on .NET Core
+
+The following APIs will always through a <xref:System.PlatformNotSupportedException> when run on .NET Core on the specified platform.
+
+This article organizes the affected API members by namespace.
+
+> [!NOTE]
+> This article is a work-in-progress. It is not a complete list of APIs that throw exceptions on .NET Core.
+
+## System
+
+| Member | Platform |
+| - | - |
+| <xref:System.AppDomain.CreateDomain%2A?displayProperty=nameWithType> | All |
+| <xref:System.AppDomain.ExecuteAssembly(System.String,System.String[],System.Byte[],System.Configuration.Assemblies.AssemblyHashAlgorithm)?displayProperty=nameWithType> | All |
+| <xref:System.Console.CapsLock?displayProperty=nameWithType> | Unix |
+| <xref:System.Console.NumberLock?displayProperty=nameWithType> | Unix |
+| <xref:System.Delegate.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)?displayProperty=nameWithType> | All |
+| <xref:System.Exception.SerializeObjectState?displayProperty=nameWithType> | All |
+| <xref:System.MarshalByRefObject.GetLifetimeService?displayProperty=nameWithType> | All |
+| <xref:System.MarshalByRefObject.InitializeLifetimeService?displayProperty=nameWithType> | All |
+| <xref:System.OperatingSystem.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)?displayProperty=nameWithType> | All |
+| <xref:System.Type.ReflectionOnlyGetType(System.String,System.Boolean,System.Boolean)?displayProperty=nameWithType> | All |
+
+## System.CodeDom.Compiler
+
+| Member | Platform |
+| - | - |
+| <xref:System.CodeDom.Compiler.CodeDomProvider.CompileAssemblyFromDom%2A?displayProperty=nameWithType> | All |
+| <xref:System.CodeDom.Compiler.CodeDomProvider.CompileAssemblyFromFile%2A?displayProperty=nameWithType> | All |
+| <xref:System.CodeDom.Compiler.CodeDomProvider.CompileAssemblyFromSource%2A?displayProperty=nameWithType> | All |
+
+## System.Collections.Specialized
+
+| Member | Platform |
+| - | - |
+| <xref:System.Collections.Specialized.NameObjectCollectionBase.%23ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)?displayProperty=nameWithType> | All |
+| <xref:System.Collections.Specialized.NameObjectCollectionBase.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)?displayProperty=nameWithType> | All |
+| o<xref:System.Collections.Specialized.NameObjectCollectionBase.OnDeserialization(System.Object)?displayProperty=nameWithType> | All |
+
+## System.Configuration
+
+| Member | Platform |
+| - | - |
+| <xref:System.Configuration.RsaProtectedConfigurationProvider?displayProperty=nameWithType> (all members) | All |
```

Comment:
Should I list the members out explicitly, or is saying "all members" sufficient?